src/main/java/io/github/cottonmc/libcd/LibCD.java

```diff
@@ -88,6 +94,15 @@ public void onInitialize() {
 			libcdNode.addChild(debugNode);
 			dispatcher.getRoot().addChild(libcdNode);
 		});
+
+		// SYSTEM TEST
```

Comment:
This should definitely be removed before release, and if we provide any stock reward types, they should go in `CDContent` instead.